hazelcast/src/main/java/com/hazelcast/spi/merge/MergingValue.java

```diff
@@ -22,20 +22,19 @@
  * @param <V> the type of the value
  * @since 3.10
  */
-public interface MergingValue<V> {
+public interface MergingValue<V> extends MergingView {
 
     /**
      * Returns the merging value in the in-memory format of the backing data structure.
      *
      * @return the merging value
      */
-    V getValue();
+    Object getRawValue();
 
     /**
      * Returns the deserialized merging value.
      *
-     * @param <DV> the type of the deserialized value
      * @return the deserialized merging value
      */
-    <DV> DV getDeserializedValue();
+    V getDeserializedValue();
```

Comment:
naming this method as `getValue` can be clearer?

Reply:
Thanks for spotting this. The idea was to implement your suggestion in https://github.com/hazelcast/hazelcast/pull/16159#discussion_r352648457 but I clearly remembered it wrong. I will change the method name.

Reply:
Done.

Reply:
And did the same with `MergingEntry#getDeserializedKey`. It's `getKey` now.